resources/monitoring/templates/prometheus-operator/cleanup-crds.yaml

```diff
@@ -1,45 +0,0 @@
-{{- if and .Values.prometheusOperator.enabled .Values.prometheusOperator.cleanupCustomResource }}
-apiVersion: batch/v1
```

Comment:
I think we still need this @a-thaler

Reply:
Are you sure? It used to be a part of the upstream chart, but not anymore. It was also not rendered on our side since cleanupCustomResource was set to `false`